### ISSUE TITLE
Feature request: Execution of file after "install" step

### ISSUE THREAD

**Issue by arkadiuszczapiewskinordea** [NordeaOSS/devtool#1]
Can we please add possibility of execution of a file after package download and unzip? 
We've got JetBrains ReSharper that needs execution of installater but doesn't require admin rights.

**Comment by holmbech** [comment]
Sounds like a good idea.
Just have in mind that this can lead to more complex installs that can have a higher frequency of failing.

**Comment by arkadiuszczapiewskinordea** [comment]
Yeah, I'm aware of it but it can be very helpful - e.g. with JetBrains ReSharper, which doesn't need admin rights, but needs to execute a "configuration" tool to add its files to Visual Studio. 

Also - it could allow to execute post-installation scripts (which can be included in packages).
